app/src/common/shared/com/igalia/wolvic/ui/views/CustomFastScroller.java

```diff
@@ -361,7 +361,7 @@ void updateScrollPosition(int offsetX, int offsetY) {
         int verticalContentLength = mRecyclerView.computeVerticalScrollRange();
         int verticalVisibleLength = mRecyclerViewHeight;
         mNeedVerticalScrollbar = verticalContentLength - verticalVisibleLength > 0
-                    && mRecyclerViewHeight >= mScrollbarMinimumRange || mNeedHorizontalScrollbar;
+                    && mRecyclerViewHeight >= mScrollbarMinimumRange || mNeedVerticalScrollbar;
```

Comment:
Good catch, this could even be a separate commit. Not sure if that was causing any actual visible bug.

Reply:
Moved to a separate commit.